modules/docs/arrow-docs/docs/docs/optics/cons/README.MD

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title: Cons
+permalink: /docs/optics/cons/
+---
+
+## Cons
+
+{:.beginner}
+beginner
+
+`Cons` provides a [Prism]({{ '/docs/optics/prism' | relative_url }}) between `S` and its head `A` and tail `S`.
```

Comment:
`tail S. It provides a convenient way to attach or detach elements to the beginning side of a structure [S].`